src/price/rates.ts

```diff
@@ -1,10 +1,11 @@
 import { ContractRunner } from 'ethers';
 import { Decimal } from '@tempusfinance/decimal';
 import { WstETH__factory } from '../typechain';
+import { STETH_RATE_PRECISION } from '../constants';
```

Comment:
You can read if from token config. No need for another constant

Reply:
Okey, yeah, we can assume rate precision is same as token precision.